src/tests/unit/services/DocumentSelectionRequestProcessor.test.ts

```diff
@@ -701,7 +700,7 @@ describe("DocumentSelectionRequestProcessor", () => {
 
 		mockYotiService.generateInstructions.mockResolvedValueOnce(HttpCodesEnum.OK);
 
-		await mockDocumentSelectionRequestProcessor.processRequest(VALID_REQUEST, "RandomF2FSessionID", encodedHeader);
+		await mockDocumentSelectionRequestProcessor.processRequest(payload, "RandomF2FSessionID", encodedHeader);
```

Comment:
Like the test above I feel like we should assert on the status here as well as these are the success tests